src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,22 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Random RANDOM = new Random();
```

Comment:
Random and ColorSupplier  shouldn't be static just private final  (the same for Lottery class)
```suggestion
    private final Random RANDOM = new Random();
```